census_data_aggregator/__init__.py

```diff
@@ -214,3 +214,108 @@ def approximate_median(range_list, design_factor=None):
 
     # Return the result
     return estimated_median, margin_of_error
+
+
+def approximate_proportion(numerator_pair, denominator_pair):
+    """
+    Calculates an estimate of the proportion and its approximate MOE
+    for the numerator and denominator arguments, where the numerator
+    represents a subset of the denominator.
+
+    Args:
+        numerator (list): a two-item sequence with a U.S. Census
+            bureau estimate and its margin of error.
+
+        denominator (list): a two-item sequence with a U.S. Census
+            bureau estimate and its margin of error.
+
+    Returns:
+        A two-item sequence containing the approximated estimate and MOE.
+
+    Examples:
+        >>> approximate_proportion((379, 1), (384, 1))
+        (0.9869791666666666, 0.008208247339752435)
+
+    """
+    # Pull out the values
+    numerator_estimate, numerator_moe = numerator_pair
+    denominator_estimate, denominator_moe = denominator_pair
+
+    # Approximate the proportion
+    proportion_estimate = 1.0 * numerator_estimate / denominator_estimate
```

Comment:
What is the reason for multiplying by 1.0 here? Doesn't the result come out the same regardless?

Reply:
It's omissible in Python 3 , but in Python 2, if division arguments are given as integers, it defaults to integer division. Happy to remove if it you prefer.

Reply:
How would you feel about substituting from __future__ import division to the top of the script to provide reverse compatibility on division of integers?

Reply:
Forgot about that one!  Done.